build/yaml/webhook/nsx-operator.yaml

```diff
@@ -0,0 +1,510 @@
+apiVersion: v1
```

Comment:
is this file generated by kube-builder? using this file, in NCP all in one yaml, are those templates still needed?